src/app/screens/Discover/websites.json

```diff
@@ -114,6 +114,12 @@
         "logo": "https://cdn.getalby-assets.com/alby-extension-website-screen/notebin-logo.png",
         "url": "https://notebin.org/"
       },
+      {
+        "title": "Pinstr",
+        "subtitle": "Social network for curating and sharing your interests",
+        "logo": "https://user-images.githubusercontent.com/100958893/238246267-ab34f64e-396a-4574-ad20-d363e5d4805a.png",
```

Comment:
Do you know how the other logos were uploaded to cdn.getalby-assets.com?

Reply:
No, I don't know how it is done. The repository maintainers upload the logo and provide the link.  P.s. That's how it's been for me for 3 PRs.

Reply:
ok, cool, just curiosity, thanks